### PR TITLE
Remove quire-starter and quire-starter-theme refs

### PR DESCRIPTION
**Pull request by geealbers**

### CODE REVIEW

**Comment by geealbers** [comment]
@Erin-Cecele I still haven't tackled the Developers, Getting Started or Netlify pages, but can you review what's here so far? This is primariyl to get rid of references to quire-starter and quire-starter-theme.

File: content/tutorial.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Follow the links below to install Quire:
 
 ## 3. Create a New Project
 
-To start a new Quire project, open your command-line shell and type `quire new my-project`. Quire will download a new starter project into a folder named “my-project” in your home directory. If you are using the Beta version of Quire, you may need to enter your GitHub username and password twice during the download process: once for the starter kit and again for the starter theme.
+To start a new Quire project, open your command-line shell and type `quire new my-project`. Quire will download a new starter project into a folder named “my-project” in your home directory.
```
Comment (Erin-Cecele) [inline content/tutorial.md:53]: Is it worth explaining what we mean by "starter project" (i.e. sample publication including content, images, and relevant metadata) or do you think it's self-explanatory?

**Review by Erin-Cecele — COMMENTED** [review]
Looks great. Just two small questions.

File: content/documentation/styles-customization.md [hunk 2]
```diff
@@ -12,15 +12,15 @@ The look and feel of your Quire publication can be customized at four different
   3. Overriding specific theme {{< q-def "templates" >}} with your own custom version
   4. Creating an entirely new Quire theme.
 
-{{< q-class "box tip" >}}  
-- The default theme installed with every new Quire project is the [Quire Starter Theme](https://github.com/gettypubs/quire-starter-theme). The README file of that repository includes complete information about the customizations available in that specific theme.
+{{< q-class "box tip" >}}
+- A default theme is installed with every new Quire project. The [README file](https://github.com/thegetty/quire/blob/main/themes/default/README.md) for that theme includes complete information about the customizations available within it.
```
[the remark this answers is not in the document]
Comment (geealbers) [reply content/documentation/styles-customization.md:16]: Yes, I think you're right that there's an issue here with how/where/why we expose these variables. Probably part of a larger project though. For instance, we could potentially identify the key variables and move them to a config yaml file for non-technical users to have more direct access to them.

File: content/tutorial.md (hunk 1 above)
Comment (geealbers) [reply content/tutorial.md:53]: good idea. I'll add this

**Pull request merged**